### ISSUE TITLE
The library, drive, or media pool is empty. (Exception from HRESULT: 0x800710D2)

### ISSUE THREAD

**Issue by yoloswaggerboy69** [TyJOrtiz/Mustastic#17]
**To Reproduce**
Open App

**Expected behavior**
the app is supposed to get the collection ready

**Desktop (please complete the following information):**
 - Windows Version [19042.964]
 - Architecture [x64]
 - Memory (16GB)

**Comment by TyJOrtiz** [comment]
Do you have a network drive in the library? This will cause an error like this to occur.

**Comment by yoloswaggerboy69** [comment]
what do you consider the library ? my windows music library is on my local disk, i use onedrive for my documents folder if that's at all relevant.
Otherwise I have an external Hard Drive but i have no traditional network drivers connected to my system

**Comment by TyJOrtiz** [comment]
Your library, as in “C:\Users\*user*\Music” or any other folder connected to the “Music” library

[cid:CA9834A1-FF45-48AE-B24B-4FFFB9F868A9]

**Comment by yoloswaggerboy69** [comment]
okay in that case, no.
No network drive in my library I'm afraid.
Any ideas, what else could be causing it ?

**Comment by TyJOrtiz** [comment]
Do you have indexing on? I can't imagine why it's not working.

**Comment by yoloswaggerboy69** [comment]
yes it's enabled for my user folder so including my music folder. which just contains 1 test mp3 file at the moment

**Comment by Tminus1** [comment]
I have this issue as well. I have my music folder's location changed from the default one and backed up on my Google Drive. If I copy and paste my music in a different folder and add that folder in the app it still doesn't show up in the library though. The "Folders" tab is the only one that's working properly.

**Comment by yoloswaggerboy69** [comment]
Not to nag or anything but I just wanted to mention, that it still isn't working for me though I discovered "open with" at least opens Mustastic but doesn't actually play any of the files I've tested. Let me know if there's a way for me to send you logs or something along those lines.

**Comment by Tminus1** [comment]
I moved all my mp3s to a different folder then back in the Music folder and this fixed it for me.

**Comment by yoloswaggerboy69** [comment]
nope sorry, didn't change anything for me :(

**Comment by TyJOrtiz** [comment]
What kind of file are you trying to open? Is it an mp3 file?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by yoloswaggerboy69** [comment]
yes it's an .mp3 file but i've tried other mp3s and wav files and none of them worked

**Comment by tophycrisp** [comment]
> Not to nag or anything but I just wanted to mention, that it still isn't working for me though I discovered "open with" at least opens Mustastic but doesn't actually play any of the files I've tested. Let me know if there's a way for me to send you logs or something along those lines.

Exact same thing happens when I "open with" or set Mustastic as the default app for audio files. The app would launch but nothing would play. Should I create a separate issue?
